webpack/scenes/ContentViews/Details/Filters/__tests__/cvModuleStreamFilter.test.js

```diff
@@ -0,0 +1,484 @@
+
+  // Nothing will show at first, page is loading
+  expect(queryByText(name)).toBeNull();
+
+  await patientlyWaitFor(() => {
+    expect(getByText(name)).toBeInTheDocument();
+    // "Select all rows"
+    expect(getByLabelText('Select all rows')).toBeInTheDocument();
+    expect(getByLabelText('bulk_actions')).toHaveAttribute('aria-expanded', 'false');
+  });
+  fireEvent.click(getByLabelText('Select all rows'));
+  fireEvent.click(getByLabelText('bulk_actions'));
+  expect(getByLabelText('bulk_actions')).toHaveAttribute('aria-expanded', 'true');
+  expect(getByLabelText('bulk_remove')).toBeInTheDocument();
+  fireEvent.click(getByLabelText('bulk_remove'));
+
+  assertNockRequest(autocompleteScope);
+  assertNockRequest(cvFilterScope);
+  assertNockRequest(cvFiltersScope);
+  assertNockRequest(cvFiltersRuleBulkDeleteScope);
+  assertNockRequest(cvRequestCallbackScope);
+
+  assertNockRequest(moduleStreamsScope, done);
+});
+
+test('Can bulk add filter rules', async (done) => {
+  const { rules } = cvFilterDetails;
+  const { name } = rules[0];
+
+  const cvFilterScope = nockInstance
+    .get(cvFilterDetailsPath)
+    .query(true)
+    .reply(200, cvFilterDetails);
+
+  const cvFiltersScope = nockInstance
+    .get(cvFiltersPath)
+    .query(true)
+    .reply(200, cvFilterFixtures);
+  const bulkAddParams = { rules_params: [{ module_stream_ids: [14] }] };
+
+  const cvFiltersRuleBulkAddScope = nockInstance
+    .put(cvBulkAddFilterRulesPath, bulkAddParams)
+    .reply(200, {});
+
+  const moduleStreamsScope = nockInstance
+    .get(moduleStreamsPath)
+    .query(true)
+    .reply(200, allModuleStreams);
+
+  const cvRequestCallbackScope = nockInstance
+    .get(cvRefreshCallbackPath)
+    .query(true)
+    .reply(200, cvFilterDetails);
+
+  const autocompleteScope = mockAutocomplete(nockInstance, autocompleteUrl);
+
+  const { getByText, queryByText, getByLabelText } =
+    renderWithRedux(withCVRoute(<ContentViewFilterDetails cvId={1} />), renderOptions);
+
+  // Nothing will show at first, page is loading
+  expect(queryByText(name)).toBeNull();
+
+  await patientlyWaitFor(() => {
+    expect(getByText(name)).toBeInTheDocument();
+    expect(getByLabelText('Select all rows')).toBeInTheDocument();
+    expect(getByLabelText('bulk_actions')).toHaveAttribute('aria-expanded', 'false');
+  });
+  fireEvent.click(getByLabelText('Select all rows'));
+  fireEvent.click(getByLabelText('bulk_actions'));
+  expect(getByLabelText('bulk_actions')).toHaveAttribute('aria-expanded', 'true');
+  expect(getByLabelText('bulk_add')).toBeInTheDocument();
+  fireEvent.click(getByLabelText('bulk_add'));
+
+  assertNockRequest(autocompleteScope);
+  assertNockRequest(cvFilterScope);
+  assertNockRequest(cvFiltersScope);
+  assertNockRequest(cvFiltersRuleBulkAddScope);
+  assertNockRequest(cvRequestCallbackScope);
+
+  assertNockRequest(moduleStreamsScope, done);
+});
+
+test('Can show affected repository tab and remove affected repos', async (done) => {
```

Comment:
We test this and the next test in PackageGroup filters test and it needs some changes based on Jeremy's PR for multipage select. We can safely remove these here cause the component for affected repository tab itself is tested in that test.